src/components/_classes/field/Field.js

```diff
@@ -5,6 +5,13 @@ export default class Field extends Component {
     if (this.noField) {
       return super.render(element);
     }
+    else if (this.isLegacyLabel && this.options.pdf) {
```

Comment:
So, this will only work with PDF forms?  If you found a generic way to solve this, then this can be pulled into the core renderer for all rendered forms.